framework/post_processors/CellVolumeIntegralPostProcessor.cc

```diff
@@ -145,6 +145,21 @@ void CellVolumeIntegralPostProcessor::Execute(const Event& event_context)
 
     value_ = ParameterBlock("", globl_integral / globl_volume);
   }
+
+  const int event_code = event_context.Code();
+  if (event_code == 32 /*SolverInitialized*/ or
+      event_code == 38 /*SolverAdvanced*/)
+  {
+    const auto& event_params = event_context.Parameters();
+
+    if (event_params.Has("timestep_index") and event_params.Has("timestamp"))
```

Comment:
Is `timestamp` the same as `time`? If so, I would recommend changing `timestep_index` to `timestamp_index` to be more clear.

Reply:
Changed `timestamp` to `time`